package.json

```diff
@@ -46,7 +46,9 @@
   },
   "homepage": "https://github.com/staltz/xstream#readme",
   "dependencies": {
-    "symbol-observable": "1.2.0"
+    "@types/globalthis": "^1.0.1",
```

Comment:
Shouldn't this go to dev dependencies? `globalthis` types are not in the puublic API of xstream, so users don't need to install it.

Reply:
Sorry I'm too late before merge. Did not want to comment without knowing if this dependency stays at all.

Reply:
Oh yeah, well noted. I'll do that in a separate commit, now it's too late to review, this has been merged.

Reply:
This is imported in `src/index.ts`. Is it the case that `tsc` inlines the dependency in a prepublish build step?

Reply:
`globalthis` is imported in runtime, but `@types/globalthis` is not, and users of the library will be importing the typescript-to-javascript transpiled code, built on prepublish.

Reply:
Ah, very good. Thanks!